.github/renovate.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "config:recommended",
+        ":semanticCommitTypeAll(chore)",
+        "helpers:pinGitHubActionDigests"
+    ],
+    "commitMessagePrefix": "deps: ",
+    "ignorePresets": [
+        ":semanticPrefixFixDepsChoreOthers"
+    ],
+    "ignorePaths": [
+        "docs/",
+        "data/",
+        ".github/"
+    ],
+    "prConcurrentLimit": 0,
+    "rebaseWhen": "behind-base-branch",
+    "dependencyDashboard": true,
+    "dependencyDashboardLabels": [
+        "type: process"
+    ],
+    "semanticCommits": "disabled",
+    "pip_requirements": {
+        "fileMatch": [
+            "*.requirements.txt",
+            "*.requirements-test.txt"
```

Comment:
Only `requirements-test.txt` should be needed, since the requirements.txt is added by default. It seems like the regex `*.` might not be needed, see https://docs.renovatebot.com/modules/manager/pip_requirements/

Reply:
deleted regex expression